spec/requests/api/v1/items_request_spec.rb

```diff
@@ -0,0 +1,101 @@
+require "rails_helper"
+
+describe "Items Request" do
+  it "can show you all of the items" do
+    create_list(:item, 10)
+
+    get "/api/v1/items"
+
+    items = JSON.parse(response.body)
+
+    expect(response).to be_success
+    expect(items.count).to eq(10)
+    expect(items[0]["name"]).to eq("Opakawagalaga Eupanifahorious")
+    expect(items[0]["description"]).to eq("Humphry Davenport The Second; Esquire, Duke of Portsholdshirelington")
+    expect(items[0]["unit_price"]).to eq(4999)
```

Comment:
can also add the following test to cover all the attributes in one line. expect(response.body).to match(/items.to_json/)